.appveyor.yml

```diff
@@ -0,0 +1,93 @@
+# Config file for automatic testing at appveyor.com
+
+version: '{branch}-{build}'
+
+build: off
+
+cache:
+  - "%LOCALAPPDATA%\\pip\\Cache"
+
+environment:
```

Comment:
Do we really want to run all these environments? We might just be duplicating the work done by our normal travis test environment already. Maybe let's just take python 2.7 and 3.6.